ext/ddtrace.stub.php

```diff
@@ -154,12 +154,12 @@ public function getLink(): SpanLink {}
         /**
          * @var SpanStack|null The parent stack, or 'null' if there is none
          */
-        public readonly SpanStack|null $parent = null;
```

Comment:
Is there a reason you removed that readonly here?

Reply:
Yes. You may notice that like [153](https://github.com/DataDog/dd-trace-php/pull/2180/files#diff-61e6285b55a2a83eebb2ae49717564d1a5f0307aee1d30d7df2bbf598826c5d1L153) defines whole class as a readonly, so it's invalid to duplicate it to property too.